pkgs/build-support/dotnet/add-nuget-deps/default.nix

```diff
@@ -24,6 +25,21 @@ let
         [ nugetDeps ]
       else if lib.isList nugetDeps then
         nugetDeps
+      else if nugetDepsIsLockFile then
```

Comment:
I feel like this might be cleaner as a separate `nugetLockFile` attribute

Reply:
Agreed, using the same attribute for both is confusing.

Reply:
Sounds good. I'll go with `nugetLockFile` in the next attempt.